tests/service/Symbols.fs

```diff
@@ -1420,7 +1420,7 @@ type Foo =
                 """
 module internal FSharp.Compiler.ParseHelpers
 
-/// The error raised by the parse_error_rich function, which is called by the parser engine
+// The error raised by the parse_error_rich function, which is called by the parser engine
```

Comment:
@DedSec256 Why this change was needed?

Reply:
This test checks that the attributes should be included in the declaration range, so XmlDoc in it seems redundant. We have separate tests with XmlDoc's and attributes in XmlDocTests.fs